src/world/CMakeLists.txt

```diff
@@ -87,6 +102,36 @@ set(WORLD_SRCS
     ${SRC_LOCAL_DIR}
 )
 
+# version-specific includes.
```

Comment:
check it out in the develop branch it's not there

Reply:
Please use latest develop for PRs. There are no version specific files in these directories...